### PR TITLE
Fix the user docs of table attributes

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Change `；` to `;` and make the user docs more readable.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: table-attributes.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ summary: 介绍 TiDB 的 `ATTRIBUTES` 使用方法。
...
     ```sql
-    alter table t [partition p ]attributes[=]'key=value[, key1=value1...]';
+    alter table t [partition p] attributes [=] 'key=value[, key1=value1...]';
     ```
 
 + 重置表或分区属性
 
     ```sql
-    alter table t [partition p ]attributes[=]default;
+    alter table t [partition p] attributes [=] default;
```
Comment (en-jin19) [inline table-attributes.md:28]: 这里的 [=] 我们还需要保留吗？@rleungx

**Review by glkappe — APPROVED** [review]
LGTM

**Review by dveeden — APPROVED** [review]
My personal preference would be to keep the `=`, but both are ok with me.

Only reviewing the statements here, not the text.

**Review by TomShawn — APPROVED** [review]
LGTM

File: table-attributes.md [hunk 2]
```diff
@@ -19,13 +19,13 @@ summary: 介绍 TiDB 的 `ATTRIBUTES` 使用方法。
 + 设置表或分区属性
```
Comment (TomShawn) [inline table-attributes.md:19]:
```suggestion
+ 设置表或分区属性 
```

File: table-attributes.md (hunk 1 above)
Comment (rleungx) [reply table-attributes.md:28]: 我觉得可以吧？

**Review by en-jin19 — APPROVED** [review]
LGTM

File: table-attributes.md [hunk 3]
```diff
@@ -16,43 +16,43 @@ summary: 介绍 TiDB 的 `ATTRIBUTES` 使用方法。
 
 表属性为 `key=value` 的形式，多个属性需要用逗号分隔。具体示例如下，其中 `t` 为所要修改的表名，`p` 为所要修改的分区名，`[]`内部为可选项。
 
-+ 设置表或分区属性
++ 设置表或分区属性
```
Comment (rleungx) [inline table-attributes.md:19]:
```suggestion
+ 设置表或分区属性
```

File: table-attributes.md [hunk 4]
```diff
@@ -73,47 +75,49 @@ alter table t partition p attributes[=]'key=value1';
...
     ```sql
-    alter table t attributes[=]'merge_option=allow';
+    ALTER TABLE t ATTRIBUTES 'merge_option=allow';
     ```
 
 + 重置某个表的属性
 
     ```sql
-    alter table t attributes[=]default；
+    ALTER TABLE t ATTRIBUTES default;
```
Comment (rleungx) [inline table-attributes.md:95]: Does `default` need to use uppercase? @dveeden

File: table-attributes.md [hunk 5]
```diff
@@ -63,6 +63,8 @@ alter table t partition p attributes[=]'key=value1';
 
 当写入或读取数据存在热点时，可以使用表属性控制 Region 合并，通过为表或分区添加 `merge_option` 属性，将其设置为 `deny` 来解决。以下介绍了两种使用场景。
 
+通过以上设置仅能处理部分热点问题。如需了解更多的热点问题处理相关内容，请参阅 [TiDB 热点问题处理](/troubleshoot-hot-spot-issues.md)。
+
```
Comment (en-jin19) [inline table-attributes.md:67]:
```suggestion
```

File: table-attributes.md [hunk 6]
```diff
@@ -73,47 +75,49 @@ alter table t partition p attributes[=]'key=value1';
```
Comment (en-jin19) [inline table-attributes.md:75]:
```suggestion
> **注意：**
>
> 通过以上设置仅能处理部分热点问题。如需了解更多的热点问题处理相关内容，请参阅 [TiDB 热点问题处理](/troubleshoot-hot-spot-issues.md)。
```

File: table-attributes.md [hunk 7]
```diff
@@ -73,47 +75,49 @@ alter table t partition p attributes[=]'key=value1';
...
     ```sql
-    alter table t partition p attributes[=]'merge_option=allow';
+    ALTER TABLE t PARTITION p ATTRIBUTES 'merge_option=allow';
     ```
 
 + 查看所有配置了 `merge_option` 属性的表或分区
 
     ```sql
-    select * from information_schema.attributes where attributes like '%merge_option%';
+    SELECT * FROM information_schema.attributes WHERE attributes LIKE '%merge_option%';
```
Comment (en-jin19) [inline table-attributes.md:113]: maybe upper cases for "attributes" here as well? @dveeden

Comment (dveeden) [reply table-attributes.md:113]: `attributes` here is the name of a column (an identifier) and not a keyword. So changing this to uppercase is not required.

File: table-attributes.md (hunk 4 above)
Comment (en-jin19) [reply table-attributes.md:95]: according to changes in https://github.com/pingcap/docs/pull/6950, the "default" here needs to use uppercase.

Comment (en-jin19) [reply table-attributes.md:95]:
```suggestion
    ALTER TABLE t ATTRIBUTES DEFAULT;
```

File: table-attributes.md [hunk 8]
```diff
@@ -73,47 +75,49 @@ alter table t partition p attributes[=]'key=value1';
...
     ```sql
-    alter table t partition p attributes[=]'merge_option=deny';
+    ALTER TABLE t PARTITION p ATTRIBUTES 'merge_option=deny';
     ```
 
 + 允许属于某个分区的 Region 被合并
 
     ```sql
-    alter table t partition p attributes[=]'merge_option=allow';
+    ALTER TABLE t PARTITION p ATTRIBUTES 'merge_option=allow';
```
Comment (en-jin19) [inline table-attributes.md:107]:
```suggestion
    ALTER TABLE t PARTITION p attributes 'merge_option=allow';
```

File: table-attributes.md [hunk 9]
```diff
@@ -73,47 +73,49 @@ alter table t partition p attributes[=]'key=value1';
...
     ```sql
-    alter table t attributes[=]default；
+    ALTER TABLE t ATTRIBUTES DEFAULT;
     ```
 
 + 禁止属于某个分区的 Region 被合并
 
     ```sql
-    alter table t partition p attributes[=]'merge_option=deny';
+    ALTER TABLE t PARTITION p ATTRIBUTES 'merge_option=deny';
```
Comment (en-jin19) [inline table-attributes.md:99]:
```suggestion
    ALTER TABLE t PARTITION p attributes 'merge_option=deny';
```

File: table-attributes.md (hunk 7 above)
Comment (en-jin19) [reply table-attributes.md:111]:
```suggestion
    SELECT * FROM information_schema.attributes WHERE ATTRIBUTES LIKE '%merge_option%';
```
have updated according to the newly updated English PR

File: table-attributes.md [hunk 10]
```diff
@@ -19,40 +19,40 @@ summary: 介绍 TiDB 的 `ATTRIBUTES` 使用方法。
...
     ```sql
-    alter table t [partition p ]attributes[=]'key=value[, key1=value1...]';
+    ALTER TABLE t [PARTITION p] ATTRIBUTES [=] 'key=value[, key1=value1...]';
     ```
 
 + 重置表或分区属性
 
     ```sql
-    alter table t [partition p ]attributes[=]default;
+    ALTER TABLE t [PARTITION p] ATTRIBUTES [=] default;
```
Comment (en-jin19) [inline table-attributes.md:28]:
```suggestion
    ALTER TABLE t [PARTITION p] ATTRIBUTES [=] DEFAULT;
```

File: table-attributes.md [hunk 11]
```diff
@@ -73,47 +73,49 @@ alter table t partition p attributes[=]'key=value1';
...
     ```sql
-    alter table t attributes[=]default；
+    ALTER TABLE t ATTRIBUTES DEFAULT;
     ```
 
 + 禁止属于某个分区的 Region 被合并
 
     ```sql
-    alter table t partition p attributes[=]'merge_option=deny';
+    ALTER TABLE t PARTITION p attributes 'merge_option=deny';
```
Comment (en-jin19) [inline table-attributes.md:99]:
```suggestion
    ALTER TABLE t PARTITION p ATTRIBUTES 'merge_option=deny';
```

File: table-attributes.md [hunk 12]
```diff
@@ -9,7 +9,7 @@ summary: 介绍 TiDB 的 `ATTRIBUTES` 使用方法。
 
 > **注意：** 
 > 
-> - 目前 TiDB 仅支持为表或分区添加 `merge_option` 属性，用于控制 Region 合并。
+> - 目前 TiDB 仅支持为表或分区添加 `merge_option` 属性，用于控制 Region 合并。通过以上设置仅能处理部分热点问题。如需了解更多的热点问题处理相关内容，请参阅 [TiDB 热点问题处理](/troubleshoot-hot-spot-issues.md)。
```
Comment (rleungx) [inline table-attributes.md:12]:
```suggestion
> - 目前 TiDB 仅支持为表或分区添加 `merge_option` 属性，用于控制 Region 合并。该属性仅能处理部分热点问题。如需了解更多的热点问题处理相关内容，请参阅 [TiDB 热点问题处理](/troubleshoot-hot-spot-issues.md)。
```

**Pull request merged**